src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6400,6 +6400,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_DoesNotOverrideBaseEqualityContract" xml:space="preserve">
     <value>'{0}' does not override expected property from '{1}'.</value>
   </data>
+  <data name="ERR_DoesNotOverrideBasePrintMembers" xml:space="preserve">
+    <value>'{0}' does not override expected method from '{1}'.</value>
```

Comment:
nit: this is the same message as ERR_DoesNotOverrideBaseEquals. It feels like this message could be consolidated across ERR_DoesNotOverrideMethodFromObject, ERR_DoesNotOverrideBaseEquals, ERR_DoesNotOverrideBaseEqualityContract, ERR_DoesNotOverrideBasePrintMembers if we made the localized symbol kind a paramter to the message.

e.g. "'{0}' does not override expected {1} from '{2}'."

Reply:
I'll re-use existing resource string. If we want to refactor further, that can be done separately.

---
In reply to: [469424954](https://github.com/dotnet/roslyn/pull/46616#discussion_r469424954) [](ancestors = 469424954)